geomstats/geometry/special_euclidean.py

```diff
@@ -36,13 +38,46 @@
                         - 1. / 362880.]
 
 
+def embed(rotation, translation, output_shape, constant=1.):
+    r"""Embed rotation, translation couples into n+1 square matrices.
```

Comment:
I think this is called the homogeneous matrix representation. Maybe rename the function and add a link?

Reply:
True